src/components/Card.js

```diff
@@ -36,7 +35,15 @@ export default function Card({
       <ScreenReaderOnly>
         <h2>Frage: </h2>
       </ScreenReaderOnly>
-      <span>{question}</span>
+      <Text>{question}</Text>
+      <ScreenReaderOnly>
+        <Text>Kategorien:</Text>
+      </ScreenReaderOnly>
+      <CategoryWrapper role="list">
+        {categories.map(category => (
+          <Category>{category}</Category>
```

Comment:
The `key` is missing here 🙂

Reply:
> The `key` is missing here 🙂

Thx :) it's there now :) and bugs are fixed, too.